github_graphql.go

```diff
@@ -4,14 +4,84 @@ import (
 	"context"
 	"encoding/base64"
 	"errors"
+	"github.com/google/go-github/v39/github"
+	"github.com/shurcooL/githubv4"
 	"regexp"
 	"strconv"
 	"time"
-
-	"github.com/google/go-github/v39/github"
-	"github.com/shurcooL/githubv4"
 )
 
+func (g *GitHubClient) listReleasesV4EnterPrice() ([]*github.RepositoryRelease, error) {
```

Comment:
you already have `g.isEnterprise` available in `GitHubClient`, so you could modify that existing list releases method instead of adding an almost identicle one

Reply:
I know, but since it's a temporary solution, the logic of the method is almost the same, but for every reference I need to use another payload for making request, so I decided just to create new one, instead of huge if-else condition everywhere